src/components/Camera.js

```diff
@@ -20,26 +40,29 @@ const Camera = (props) => {
 
       canvasRef.current.width = videoWidth;
       canvasRef.current.height = videoHeight;
-      
-      props.onEstimate(video); 
+
+      props.onEstimate(video);
     }
   }
 
+  /**
+   * Handles animation frames.
+   */
   function onAnimate() {
     detect();
     requestAnimationFrame(onAnimate);
   }
 
   useEffect(() => {
     onAnimate();
-  }, [])
-  
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Thanks - I plan to create fix the warning in a follow-up commit. So far it's inherited from a quick hack to make sure useEffect does not re-entry